### PR TITLE
WFCORE-1051 Worker's attributes of IO subsystem are not documented pr…

### PR DESCRIPTION
**Pull request by ctomc**
…operly

### CODE REVIEW

File: io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties [hunk 1]
```diff
@@ -3,18 +3,16 @@ io.add=Operation Adds subsystem
...
+io.worker.io-threads="Specify the number of I/O threads to create for the worker.  \
+  If not specified, a default will be chosen, which is calculated by cpuCount * 2"
+io.worker.task-keepalive=Specify the number of milliseconds to keep non-core task threads alive.
 io.buffer-pool=Defines buffer pool
 io.buffer-pool.add=Adds new buffer pool
-io.buffer-pool.remove=Removes buffer -pol
-io.buffer-pool.buffers-per-slice=How many buffers per slice
-io.buffer-pool.buffer-size=How big is the buffer
-io.buffer-pool.direct-buffers=Does the buffer pool use direct buffers
+io.buffer-pool.remove=Removes buffer ppol
```
Comment (bstansberry) [inline io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties]: s/ppol/pool

File: io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties [hunk 2]
```diff
@@ -3,18 +3,16 @@ io.add=Operation Adds subsystem
...
+io.buffer-pool.buffers-per-slice=How many buffers per slice, if not set value optimal buffer is calculated avaliable RAM resources in your system.
```
Comment (bstansberry) [inline io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties]: s/value optimal buffer/optimal value/g
s/avaliable/available

File: io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties [hunk 3]
```diff
@@ -3,18 +3,16 @@ io.add=Operation Adds subsystem
...
+io.buffer-pool.buffer-size=The size of each buffer slice in bytes, if not set value optimal buffer is calculated avaliable RAM resources in your system.
```
Comment (bstansberry) [inline io/subsystem/src/main/resources/org/wildfly/extension/io/LocalDescriptions.properties]: s/value optimal buffer/optimal value/g
s/avaliable/based on available/g   (note the typo in 'avaliable')

**Comment by ctomc** [comment]
fixed, tnx

**Pull request merged**